spec/requests/versioned_file_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "Versioned File requests" do
+  before do
+    allow(Cocina).to receive(:find).and_return(Cocina.new(public_json))
```

Comment:
Should the test also make sure we're retrieving the "right" version of the cocina?

Reply:
Done.